.gitignore

```diff
@@ -18,4 +18,4 @@ venv
 /bin
 /pyvenv.cfg
 /webpack-stats.json
-/euth_wagtail/settings/local.py
+/euth_wagtail/settings/local.py
```

Comment:
There is no need to modify this file. Also `.editorconfig` says that there should always be a newline.